Mod/Villains/Leviathan/Cards/TorrentialDownpourCardController.cs

```diff
@@ -0,0 +1,49 @@
+﻿using Handelabra.Sentinels.Engine.Controller;
+using Handelabra.Sentinels.Engine.Model;
+using System.Collections;
+
+namespace Jp.ParahumansOfTheWormverse.Leviathan
+{
+
+    // Whenever a player would draw a card, Leviathan deals that player's character card 1 cold damage. This card is indestructible
+
+    public class TorrentialDownpourCardController : CardController
+    {
+        public TorrentialDownpourCardController(Card card, TurnTakerController controller) : base(card, controller)
+        {
+            //This card is indestructible
+            base.AddThisCardControllerToList(CardControllerListType.MakesIndestructible);
```

Comment:
I think this is the 'ask method' stuff and you need to implement AskIfCardIsIndestructible.